### ISSUE TITLE
api: cve: I got two different result of the same `cve` requests.

### ISSUE THREAD

**Issue by grinrag** [RedHatInsights/vmaas#274]
I tried to compare the responses of the same CVE GET request executed several times. I have a result when the responses are different.
STR:
1) execute `time curl -H "accept: application/json" -X GET --globoff  http://127.0.0.1:8080/api/v1/cves/CVE-2017-\\d+ > /tmp/one.out`

2) execute the same request the second time, but redirect the output to another file `time curl -H "accept: application/json" -X GET --globoff  http://127.0.0.1:8080/api/v1/cves/CVE-2017-\\d+ > /tmp/two.out`

3) Compare results `diff /tmp/one.out /tmp/two.out`. No difference as expected.

4) Now execute the same request the third time and at the same moment execute the another request in parallel `time curl -H "accept: application/json" -X GET --globoff  http://127.0.0.1:8080/api/v1/cves/CVE-2017-\\d+ > /tmp/three.out`
*AND* `time curl -H "accept: application/json" -X GET --globoff  http://127.0.0.1:8080/api/v1/cves/CVE-2017-5\\d+`

5) compare `one.out` and `three.out`. They are different even in size. Executing the third request takes less time than first or second

**Comment by ggainey** [comment]
OK, I can reproduce - running the "http://127.0.0.1:8080/api/v1/cves/CVE-2017-5\\d+" cmd in a loop while the "http://127.0.0.1:8080/api/v1/cves/CVE-2017-\\d+" was executing did the trick.

If you run one.out and three.out through python -m json.tool and diff the results, you will see that the difference is that one.out contains cwe_lists for each erratum, but three does not:

```
diff /tmp/one.json /tmp/three.json
5,7c5
<             "cwe_list": [
<                 "CWE-264"
<             ],
---
>             "cwe_list": [],
18,20c16
<             "cwe_list": [
<                 "CWE-264"
<             ],
---
>             "cwe_list": [],
31,33c27
<             "cwe_list": [
<                 "CWE-119"
<             ],
---
>             "cwe_list": [],
44,46c38
<             "cwe_list": [
<                 "CWE-20"
<             ],
---
>             "cwe_list": [],
57,59c49
<             "cwe_list": [
<                 "CWE-264"
<             ],
---
```

Not sure what the problem is yet, but this should be a Big Hint...

**Comment by ggainey** [comment]
Fixed by #282
